R/loadData.R

```diff
@@ -0,0 +1,29 @@
+#' These functions are to pre-load data for some cities so they can be easily visualized
+
+# how do i access functions in other files? should this stuff be in data-raw.R or smthin?
+# source("sample_people.R")
+
+#' Function to pre-load the population of Boston Massachusetts
+
+bostonSample <- create_dots(filter_data_wide(get_data_wide(), "Massachusetts", "Suffolk"), num_people = 100)
```

Comment:
I think these will be easier to read if you write it using pipes. Also, you don't need to run get_data_wide() for each sample individually. At the top, you can just run data_wide <- get_data_wide() and then modify that object for each sample. For example, the first one would look like:

```
boston_sample <- data_wide %>%
     filter_data_wide("Massachusetts", "Suffolk") %>%
     create_dots()
```
Also, if you are using num_people = 100, you don't need to specify that because that is the default